bin/ycsb.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/sh
```

Comment:
I'd love to stick to POSIX, but I am familiar with a static analizer for bash and I don't know of one for POSIX. I expect that without one we'll drift out of compliance pretty quick.